driver/terraform.go

```diff
@@ -96,7 +96,7 @@ func NewTerraform(config *TerraformConfig) (*Terraform, error) {
 		log.Printf("[ERR] (driver.terraform) init client type '%s' error: %s", config.ClientType, err)
 		return nil, err
 	}
-	if env := config.Task.Providers.Env(); len(env) > 0 {
+	if env := config.Task.Env; len(env) > 0 {
```

Comment:
Not sure if there is anything wrong here, but wanted to draw attention to it in case. 
Do the env settings from `config.Task.Providers.Env()` just not matter here anymore?

Reply:
The `providers.Env()` is merged by the caller in the changes above within `controller/controller.go`.

However after running into a different but where tfexec doesn't inherit the os env which includes `PATH` used for `git` within Terraform init, I'll be moving some of the merging around now that there are 3 environments: task, providers, and os.

Reply:
Oh... I think I see it now on line 242 above. Cool. 
Sorry for the red herring.